google-cloud-bigtable-emulator-core/src/main/java/com/google/cloud/bigtable/emulator/core/EmulatorController.java

```diff
@@ -93,11 +93,13 @@ public synchronized boolean isRunning() {
   }
 
   /** Starts the emulator process and waits for it to be ready. */
-  public synchronized void start() throws IOException, TimeoutException, InterruptedException {
+  public synchronized void start(int port)
```

Comment:
I think there are a few places that you need to update as well:

https://github.com/googleapis/java-bigtable/blob/main/google-cloud-bigtable-emulator/src/main/java/com/google/cloud/bigtable/emulator/v2/BigtableEmulatorRule.java#L53

 and https://github.com/googleapis/java-bigtable/blob/main/google-cloud-bigtable-emulator/src/main/java/com/google/cloud/bigtable/emulator/v2/Emulator.java#L61

depending on how they're creating the emulator?